.dd-ci/ci-app-spec.json

```diff
@@ -0,0 +1,23 @@
+[
+  "test.suite",
+  "test.name",
+  "test.status",
+  "test.framework",
+  "test.type",
+  "ci.provider.name",
+  "ci.pipeline.id",
+  "ci.pipeline.name",
+  "ci.pipeline.number",
+  "ci.pipeline.url",
+  "ci.stage.name",
+  "ci.job.name",
+  "ci.job.url",
+  "ci.workspace_path",
+  "git.repository_url",
+  "git.commit.sha",
+  "git.branch",
+  "git.tag",
+  "git.commit.author.email",
+  "git.commit.author.name",
+  "git.commit.message"
+]
```

Comment:
I didn't quite get what this file does or when it is used...?

Reply:
we keep a feature tracking of tracers for CI App. This feature tracking is generated automatically based on this info. The idea behind is it is easier to keep information updated when it is closer to where it is relevant. Ideally, if possible, these tags are generated automatically (kind of like a snapshot), so you don't even have to worry about it. 

There's a github action in this repo: https://github.com/DataDog/datadog-ci-spec/blob/main/.github/workflows/feature-track.yml that reads from each individual tracer's repo and generates a HTML that is published via github pages. The result can be seen in https://datadoghq.dev/datadog-ci-spec

Reply:
Thanks for the context 🙇